### DISCUSSION TITLE
How To Install Laravel On EasyEngine

### DISCUSSION THREAD

**Discussion by MiguelEmmara-ai** [EasyEngine/easyengine#1704] [Show and tell]
There is not a lot of tutorial on how to setup/install laravel app on a server running EasyEngine
So here I made one :)

*Side Note, All the 'example.com' and 'examplecom' must be change to your domain name!

Full Tutorial
https://github.com/MiguelEmmara-ai/EasyEngineLaravel

Thank You 👍

**Comment by marcoandrei** [comment]
Very good, Miguel. Thanks for sharing! (Do you speak Portuguese or Spanish in Jakarta?)
